### ISSUE TITLE
Use the dropout layer only for training

### ISSUE THREAD

**Issue by BoboyeOkeya** [BoboyeOkeya/Physics_Informed_Neural_Network#4]
Turns out that it is advisable to only use the dropout layers during training, in order to prevent overfitting.

However, when it comes to predictions, the dropout layers should not  be included in order to use the full power of the neural network
